zkevm-circuits/src/witness.rs

```diff
@@ -0,0 +1,14 @@
+#![allow(missing_docs)]
```

Comment:
Can't you simply add the minimal documentation needed?

Reply:
add basic docs except Rw struct which needs a lot of work..

Reply:
I think if you make that PR and split the stuff you need to also add anything that is required TBH.

Reply:
No there was no docs at least i did not make things worse. Tbh at least now i have no plan to add docs for rw